src/axom/inlet/examples/arrays.cpp

```diff
@@ -0,0 +1,28 @@
+#include <iostream>
+#include <unordered_map>
+
+#include "axom/inlet.hpp"
+
+void example() {

```

Comment:
There is no need for the example function.  Move it to main().  Also return 0 to avoid a warning at the end of main.

Add some comments since this is an example for the users.  For example, "Initialize Inlet" "Parse example input file" "Add verifier that array was found" (also check some of the values inside).